### PR TITLE
Fix query params being empty

### PR DESCRIPTION
**Pull request by wiesson**
Fixes https://github.com/sveltejs/kit/issues/631

The query params are not parsed correctly, because `URL` does not return `query` - instead it returns the desired searchParams!

Tested here - https://sveltekit-vercel.vercel.app/queryparams?q=pizza

### CODE REVIEW

File: .changeset/fluffy-carpets-confess.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@sveltejs/adapter-vercel': minor
```
Comment (dummdidumm) [inline .changeset/fluffy-carpets-confess.md:2]: Needs to be a patch release

Comment (wiesson) [reply .changeset/fluffy-carpets-confess.md]: Fixed!

**Comment by wiesson** [comment]
The previous job was successful, I could imagine that somekind of race condition happened because I force pushed to the branch 🤷‍♂️

**Comment by benmccann** [comment]
If this is broken, then I'm guessing `adapter-node` might be as well:

https://github.com/sveltejs/kit/blob/80241789644ca2add2fd03bc2a49e3c91689d472/packages/adapter-node/src/server.js#L40

Strange that this would only be reported now...

**Comment by benmccann** [comment]
Ah, nevermind. `adapter-node` calls `parse` which returns a legacy URL which defines `query`. We should probably change `adapter-node` as well so that it doesn't use the legacy URL creation method

**Comment by wiesson** [comment]
Should I include that in this PR as well or create a new one?

**Comment by benmccann** [comment]
I've updated this PR

**Pull request merged**